src/Sequence.cpp

```diff
@@ -62,7 +62,7 @@ Sequence::Sequence(
 Sequence::Sequence(
    const Sequence &orig, const SampleBlockFactoryPtr &pFactory)
 :  mpFactory(pFactory),
-   mSampleFormat(orig.mSampleFormat),
+   mSampleFormats{ orig.mSampleFormats },
```

Comment:
there should be two args to mSampleFormats? otherwise, m_Stored remains undefined?